google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/ConnectionOptions.java

```diff
@@ -856,6 +877,18 @@ private static Integer parseIntegerProperty(String propertyName, String value) {
     return null;
   }
 
+  /**
+   * Returns an instance of OpenTelemetry. If OpenTelemetry object is not set
+   * then GlobalOpenTelemetry will be used as fallback.
+   */
+  OpenTelemetry getOpenTelemetry() {
```

Comment:
nit: annotate this method with `@NonNull` to indicate to callers that it will never return a null value.

Reply:
Fixed.